src/libraries/System.Private.Xml.Linq/tests/TreeManipulation/SaveWithFileName.cs

```diff
@@ -256,11 +256,6 @@ private static string StripOffXmlDeclaration(string s)
             return s;
         }
 
-        private static string NormalizeNewLines(string s)
-        {
-            s = s.Replace("\n", "");
-            s = s.Replace("\r", "");
-            return s;
-        }
+        private static string NormalizeNewLines(string s) => s.ReplaceLineEndings("");
```

Comment:
Would `s.ReplaceLineEndings()` work the same?

Reply:
nvm, I just read the docs says it will replace with Environment.NewLine but I personally don't find it particularly obvious (I was expecting to remove them)